### PR TITLE
[hapanel] Fix SAT findings

### PR DESCRIPTION
**Pull request by wborn**
When this PR and #18 are merged the build succeeds without having to use `-DskipChecks`

### CODE REVIEW

File: org.openhab.ui.habpanel/pom.xml [hunk 1]
```diff
@@ -15,116 +15,17 @@
...
         <dependency>
             <groupId>org.openhab.core.bundles</groupId>
             <artifactId>org.openhab.core</artifactId>
             <version>${ohc.version}</version>
         </dependency>
         <dependency>
-            <groupId>org.openhab.core.bundles</groupId>
-            <artifactId>org.openhab.core.ui.dashboard</artifactId>
-            <version>${ohc.version}</version>
+            <groupId>org.openhab.ui</groupId>
```
Comment (martinvw) [inline org.openhab.ui.habpanel/pom.xml]: Do you really need to cause merge problems with my PR 😛

Comment (wborn) [reply org.openhab.ui.habpanel/pom.xml]: I think it's just POM changes that may conflict. If your changes are better just replace the whole POM with yours. :-)

**Comment by martinvw** [comment]
The rest looks fine btw!

**Comment by ghys** [comment]
Thanks to you both ;)

**Pull request merged**